ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/bgp_peer_groups.schema.yml

```diff
@@ -0,0 +1,126 @@
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  bgp_peer_groups:
+    documentation_options:
+      filename: Fabric Variables
+      table: BGP Peer Groups
+    description: |
+      Leverage an Arista EOS switch to generate the encrypted password using the correct peer group name.
+      Note that the name of the peer groups use '-' instead of '_' in EOS configuration.
+    type: dict
+    keys:
+      ipv4_underlay_peers:
+        type: dict
+        keys:
+          name:
+            type: str
+            default: IPv4-UNDERLAY-PEERS
+            description: Name of peer group.
+          password:
+            type: str
+            description: Encrypted password.
+          structured_config:
+            type: dict
+            description: Custom structured config added under router_bgp.peer_groups.<name> for eos_cli_config_gen
+      mlag_ipv4_underlay_peer:
+        type: dict
+        keys:
+          name:
+            type: str
+            default: MLAG-IPv4-UNDERLAY-PEER
+            description: Name of peer group.
+          password:
+            type: str
+            description: Encrypted password.
+          structured_config:
+            type: dict
+            description: Custom structured config added under router_bgp.peer_groups.<name> for eos_cli_config_gen
+      evpn_overlay_peers:
+        type: dict
+        keys:
+          name:
+            type: str
+            default: EVPN-OVERLAY-PEERS
+            description: Name of peer group.
+          password:
+            type: str
+            description: Encrypted password.
+          structured_config:
+            type: dict
+            description: Custom structured config added under router_bgp.peer_groups.<name> for eos_cli_config_gen
+      evpn_overlay_core:
+        type: dict
+        keys:
+          name:
+            type: str
+            default: EVPN-OVERLAY-CORE
+            description: Name of peer group.
+          password:
+            type: str
+            description: Encrypted password.
+          structured_config:
+            type: dict
+            description: Custom structured config added under router_bgp.peer_groups.<name> for eos_cli_config_gen
+      mpls_overlay_peers:
+        type: dict
+        keys:
+          name:
+            type: str
+            default: MPLS-OVERLAY-PEERS
+            description: Name of peer group.
+          password:
+            type: str
+            description: Encrypted password.
+          structured_config:
+            type: dict
+            description: Custom structured config added under router_bgp.peer_groups.<name> for eos_cli_config_gen
+      rr_overlay_peers:
+        type: dict
+        keys:
+          name:
+            type: str
+            default: RR-OVERLAY-PEERS
+            description: Name of peer group.
+          password:
```

Comment:
You could use a def / ref for this block to make the file smaller given that we repeat it a lot of times

Reply:
Well, the defaults and descriptions are not consistent...